Workouts/HomeView.swift

```diff
@@ -25,13 +25,15 @@ struct HomeView: View {
         return NavigationView {
             List {
                 if featuredWorkout != nil {
-                    VStack(alignment: .leading) {
-                        Text("Your latest workout 🏅")
-                            .padding(.vertical)
-                            .font(.system(size: 21, weight: .medium))
-                        FeaturedWorkout(workout: featuredWorkout!)
-                            .padding(.bottom)
-                    }
+                    NavigationLink(destination: WorkoutDetail(workout: featuredWorkout!)) {
+                        VStack(alignment: .leading) {
+                            Text("Your latest workout 🏅")
+                                .padding(.vertical)
+                                .font(.system(size: 21, weight: .medium))
+                            FeaturedWorkout(workout: featuredWorkout!)
+                                .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+                        }
+                    }.padding(.trailing, -22.0) // <-- hide that dang little arrow icon
```

Comment:
Haha nice fix 😛. Hopefully, as SwiftUI evolves, Apple will come up with easy ways to deal with little annoyances like this

Reply:
Agreeeed, kind of a hack, but it doesn't seem like there is a simple way to say - hey, I don't want an icon 😆